src/app/core/components/navigation/style.scss

```diff
@@ -29,9 +29,15 @@ header {
   .km-header-logo {
     display: block;
     height: 30px;
-    margin-left: variables.$content-padding - 10px;
+    margin: 0 (variables.$content-padding - 10px);
+    padding-right: 160pt;
```

Comment:
IMO, remove `160pt` 
- It is not required + using of different unit `pt` which is not used anywhere in application
- increase width to `210px` fill fix the issue

After adding below fix https://github.com/kubermatic/dashboard/pull/4503/files#r884792566

Note:
This will make design align with actual image reported in issue: [Image](https://github.com/kubermatic/dashboard/issues/4458#issue-1230721777)